addons/nametags/XEH_postInit.sqf

```diff
@@ -45,3 +45,12 @@ GVAR(showNamesTime) = -10;
 
 // civilians don't use military ranks
 ["CIV_F", ["","","","","","",""]] call FUNC(setFactionRankIcons);
+
```

Comment:
Maybe add a way to disable, don't think it needs a setting.
We tend to use base vanilla units and just change the uniforms, so this doesn't really make sense for us, and would probably just be confusing
Does CSAT = russian and greek = german make sense?

```suggestion

if (missionNamespace getVariable [QGVAR(skipCustomRankIcons), false]) exitWith {};
```

Reply:
Couldn't you just overwrite the civs with whatever icons you want instead by using the same function?

Reply:
civs are fine, I just want to skip the for loop

Reply:
The Russian ranks and Iranian ranks are somewhat similar, the Chinese stand out somewhat in that they used downward point chevrons mostly. I don't know exactly how to reconcile in-game lore and geopolitics with the ranks, but if NATO gets all the American ranks, then maybe CSAT should get the Chinese ones? It's a little problematic that these nationally diverse alliances (CSAT/NATO) are one faction in game.

As for the AAF getting the German ranks; the AAF is mostly NATO-built force, and they have a bunch of equipment from all over NATO, so maybe they should just get NATO ranks.

When it comes to the new ranks, I'm not fussed as to who gets what. Just tell me what you need. If need be I'll happily add in more ranks for different nations and/or create some separate ranks for the AAF, FIA. The incoming Livonians should probably use the NATO ranks.

Reply:
Make some ranks for the NVA. :)
They'd be used in GM, and they are simple.